dashboard/app/views/home/index.html.haml

```diff
@@ -5,6 +5,10 @@
 
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
+- if current_user && current_user.show_race_interstitial?
+  - current_user.race_interstitial_shown = true
+  - current_user.save
```

Comment:
@ashercodeorg or @wjordan might know better than I do what the specific conditions are, but in this case, I seemed to need to explicitly call save in order to get it to persist.

I think the terms interstitial makes a user update call when you submit the checkbox, which handles the saving automatically, updating terms_of_service_version. So you probably didn't need to do any explicit saving there.